src/main/java/duke/TaskList.java

```diff
@@ -0,0 +1,28 @@
+package duke;
+
+import java.util.ArrayList;
+
+public class TaskList {
+    private static ArrayList<Task> todoList;
```

Comment:
You could rename this to tasks to keep it consistent with the coding standards provided by prof!

Reply:
this could also be instance level (non-static) rather than class level! I think it would be easier to keep track of the collection this way.